_pages/en/gsoc-ideas.md

```diff
@@ -272,13 +272,72 @@ And, if possible, an *easy, medium or hard* rating of each project.
             <td>Medium</td>
             <td><a href="https://developersitalia.slack.com/messages/C7VPAUVB3">#design-devel</a></td>
         </tr>
+       <tr role="row" class="even">
+            <td tabindex="0">API</td>
+            <td>
+                <b>Port the <i>gen-api-models</i> tool to OpenAPI v3</b><br /> 
+                This proposal involves a set of tools used across the “IO” project. 
+                In particular, the <i>gen-api-models</i> tool allows to generate TypeScript definitions of OpenAPI specs. 
+                As such, this proposal requires adapting the existing tool to support the newer 
+                <a href="https://docs.italia.it/italia/piano-triennale-ict/lg-modellointeroperabilita-docs/">Interoperability Guidelines</a>. 
```

Comment:
dovremmo tradurre le LG in inglese per renderlo accettabile?

Reply:
Dipende dagli interessati. Io lascerei questo link e, a fronte di tante richieste da stranieri, vediamo come procedere.

Reply:
Potremmo anche dire - semplicemente - che vogliamo supportare OpenAPI 3 e il lavoro in corso verso OAS3.1 senza fare riferimento direttamente alle LG.